src/components/RunGraph.vue

```diff
@@ -39,5 +42,7 @@
     start(stage.value)
   })
 
-  onUnmounted(() => stop())
+  onBeforeUnmount(() => {
```

Comment:
worth calling out that changing this to `onBeforeUnmount` is important because it ensures the correct order of setup -> start -> stop. Using `onUnmounted` meant the `stop` method would get called out of order when vue recycles the component.